src/main/java/io/jenkins/plugins/jfrog/JfStep.java

```diff
@@ -294,4 +321,43 @@ public boolean isApplicable(Class<? extends AbstractProject> jobType) {
             return true;
         }
     }
+
+    String getJfrogCliVersion(Launcher.ProcStarter launcher) throws IOException, InterruptedException {
+        if (this.currentCliVersion != null && !this.currentCliVersion.isEmpty()) {
+            return this.currentCliVersion;
+        }
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
```

Comment:
Should be closed. Please use try-with-resources
```suggestion
        try (ByteArrayOutputStream outputStream = new ByteArrayOutputStream()) {
```